docs/HowTo/Get-Started/Run-Teku.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Running Teku
+---
+
+# Start Teku
+
+**Prerequisites**:
+
+* [Teku installed](Install-Binaries.md)
+* [Validators keystores] and [password files].
+
+You can run Teku as a beacon chain client and validator in a single process, or as separate
+processes.
+
+We recommend you run the beacon chain client and validator as a [single process] if they are to run
+on the same machine.
+
+## Start the clients in a single process
+
+Start the beacon chain client and validator as a single process by specifying the validator options
+with the [`teku`](../../Reference/CLI/CLI-Syntax.md#options) command
+
+!!! example
+
+    ```
+    teku --network=medalla --eth1-endpoint=http://localhost:8545 \
+    --validator-keys=validator/keys/validator_888eef.json:validator/passwords/validator_888eef.txt \
+    --rest-api-enabled=true --rest-api-docs-enabled=true \
+    --metrics-enabled
+    ```
+
+Use the [`--validator-keys`](../../Reference/CLI/CLI-Syntax.md#validator-keys) option to specify
+the directory or file to load the encrypted keystore file(s) and associated password file(s) from.
+
+## Run the clients separately
+
+Validators need to connect to a beacon chain client to publish attestations or propose blocks. The
+beacon node requires internet access, but the connected validators can run on machines without
+internet access.
+
+### Start the beacon chain client
+
+Run Teku as a beacon chain client.
+
+!!! example
+
+    ```bash
+    teku --network=medalla --eth1-endpoint=http://localhost:8545 \
+    --metrics-enabled --rest-api-enabled
+    ```
+
+Specify [`--rest-api-enabled`](../../Reference/CLI/CLI-Syntax.md#rest-api-enabled) to allow
+validators to connect to the beacon node.
+
+!!! warning
+    Do not pass the validator keys as a command line option to both the beacon chain client and
+    validator client. This can cause a [slashable offence].
+
+By default, [validator clients] can connect to the beacon node at `http://127.0.0.1:5051`.
+Use the [`--rest-api-interface`](../../Reference/CLI/CLI-Syntax.md#rest-api-interface)
+and [`--rest-api-port`](../../Reference/CLI/CLI-Syntax.md#rest-api-port) options to update the
+address.
+
+### Start the validator
+
+To run a validator, connect to a [running beacon node].
+
+Use the [`validator-client`](../../Reference/CLI/CLI-Subcommands.md#validator-client-vc) or
+[`vc`](../../Reference/CLI/CLI-Subcommands.md#validator-client-vc) subcommand to run a Teku as a
+validator.
+
+!!! example
+
+    ```
+    teku validator-client --network=medalla --beacon-node-api-endpoint=http://127.0.0.1:5051 \
```

Comment:
Given that we're recommending using the separate validator-client only when on a different box, `127.0.0.1` may not be a great example URL.